guides/keras_cv/classification_with_keras_cv.py

```diff
@@ -0,0 +1,688 @@
+    image = tf.cast(image, tf.float32)
+    image = resizing(image)
+    return resizing(image), tf.one_hot(label, num_classes)
+
+
+train_dataset = train_dataset.shuffle(10 * BATCH_SIZE).map(
+    preprocess_inputs, num_parallel_calls=AUTOTUNE
+)
+train_dataset = train_dataset.batch(BATCH_SIZE)
+
+images = next(iter(train_dataset.take(1)))[0]
+keras_cv.visualization.plot_image_gallery(images, value_range=(0, 255))
+
+"""
+Meow!
+
+Next let's construct our model:
+"""
+
+backbone = keras_cv.models.EfficientNetV2Backbone.from_preset(
+    "efficientnetv2_b0_imagenet",
+)
+"""
+The use of imagenet in the preset name indicates that the backbone was
+pretrained on the ImageNet dataset.
+This is called transfer learning, and results in a more efficient fitting of the
+model to our new dataset.
+Transfer learning models fit in fewer epochs, and often achieve better final
+results.
+
+Next lets put together our classifier:
+"""
+
+model = keras.Sequential(
+    [
+        backbone,
+        keras.layers.GlobalMaxPooling2D(),
+        keras.layers.Dropout(rate=0.5),
+        keras.layers.Dense(2, activation="softmax"),
+    ]
+)
+model.compile(
+    loss="categorical_crossentropy",
+    optimizer=tf.optimizers.SGD(learning_rate=0.01),
+    metrics=["accuracy"],
+)
+
+"""
+Here our classifier is just a simple `keras.Sequential`.
+All that is left to do is call `model.fit()`!
+"""
+
+model.fit(train_dataset)
+
+
+"""
+Let's look at how our model performs after the fine tuning!
+"""
+
+predictions = model.predict(np.expand_dims(image, axis=0))
+
+classes = {0: "cat", 1: "dog"}
+print("Top class is:", classes[predictions[0].argmax()])
+
+"""
+Awesome!  Looks like the model correctly classified the image.
+"""
+
+"""
+# Train a Classifier from Scratch
+
+![](https://storage.googleapis.com/keras-nlp/getting_started_guide/prof_keras_advanced.png)
+
+Now that we've gotten our hands dirty with classification, let's take on one
+last task: training a classification model from scratch!
+A standard benchmark for image classification is the ImageNet dataset, however
+due to licensing constraints we will use the CalTech 101 image classification
+dataset in this tutorial.
+While we use the simpler CalTech 101 dataset in this guide, the same training
+template may be used on ImageNet to achieve state of the art scores.
+
+Let's start out by tackling data loading:
+"""
+NUM_CLASSES = 101
+# Change epochs to 100~ to fully train.
+EPOCHS = 1
+
+
+def package_inputs(image, label):
+    return {"images": image, "labels": tf.one_hot(label, NUM_CLASSES)}
+
+
+train_ds, eval_ds = tfds.load(
+    "caltech101", split=["train", "test"], as_supervised="true"
+)
+train_ds = train_ds.map(package_inputs, num_parallel_calls=tf.data.AUTOTUNE)
+eval_ds = eval_ds.map(package_inputs, num_parallel_calls=tf.data.AUTOTUNE)
+
+train_ds = train_ds.shuffle(BATCH_SIZE * 16)
+
+train_ds = train_ds.ragged_batch(BATCH_SIZE)
+eval_ds = eval_ds.ragged_batch(BATCH_SIZE)
```

Comment:
These lines deserve some explanation. I don't think the average user will know what's going on here

Shuffling can be helpful for more diverse batches, why ragged?

Reply:
I explain the ragged aspect in the RandomCropAndResize section - is that sufficient?

Reply:
added another phrase